cluster/node.go

```diff
@@ -54,6 +54,7 @@ type Options struct {
 	IsWebsocket    bool
 	TSLCertificate string
 	TSLKey         string
+	OnUnregister   func(string)
```

Comment:
Prefer to use `UnregisterCallback` or `UnregisterHook`. And I think the callback parameter should be node information instead of only server address.